internal/provider/data_source.go

```diff
@@ -79,11 +76,17 @@ func dataSource() *schema.Resource {
 			},
 
 			"body": {
-				Type:     schema.TypeString,
-				Computed: true,
-				Elem: &schema.Schema{
-					Type: schema.TypeString,
-				},
+				Description: "The raw body of the HTTP response. " +
+					"**NOTE**: This is deprecated, use `response_body` instead.",
+				Type:       schema.TypeString,
+				Computed:   true,
+				Deprecated: "Use response_body instead",
```

Comment:
lets keep 

```golang
				Elem: &schema.Schema{
					Type: schema.TypeString,
				},
```

in the `body` and `request_body`.    I do know in the upstream provider [here](https://github.com/hashicorp/terraform-provider-http/blob/main/internal/provider/data_source_http.go#L208-L209) the Elem isn't set but later on they retun the datasrource value as a type.  if its truly not necessary, we can clean up all references in another PR

Reply:
Sure, done